tests/unit/test_index_html_page.py

```diff
@@ -118,6 +118,34 @@ def test_get_html_response_no_head(url):
     ]
 
 
+def test_get_html_response_dont_log_clear_text_password(caplog):
+    """
+    `_get_html_response()` shouldn't log the full index's URL includoing the
```

Comment:
Typo: including

Also, this could be a little clearer. How about: "`_get_html_response()` should redact the password from the index URL in its DEBUG log message."